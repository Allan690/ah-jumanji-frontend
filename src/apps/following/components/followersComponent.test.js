import React from 'react';
import { shallow } from 'enzyme';

// Import container
import Follow from './followersComponent';

describe('Follow Container', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Follow debug />);

    expect(component).toMatchSnapshot();
  });
});
