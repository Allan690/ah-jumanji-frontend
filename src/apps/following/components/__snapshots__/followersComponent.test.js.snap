// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Follow Container should render correctly in "debug" mode 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
